articles/azure-monitor/app/tutorial-asp-net-core.md

```diff
@@ -0,0 +1,525 @@
+
+    ![The Performance screen of Application Insights displays with the toggle between Server and Browser highlighted along with the list of HTTP calls received by the application.](./media/asp-net-core/server-performance.png "Server performance HTTP calls")
+
+4. Select an Operation from the table, and choose to drill into a sample of the request.
+
+    ![The Performance screen displays with a POST operation selected, the Drill into samples button is highlighted and a sample is selected from the suggested list.](./media/asp-net-core/select-operation-performance.png "Drill into an operation")
+
+5. The End-to-end transaction displays for the selected request. In this case, a review was created including an image, thus it includes calls to Azure Storage, the Language Service (for sentiment analysis), as well as database calls into SQL Azure to persist the review. In this example, the first selected Event displays information relative to the HTTP POST call.
+
+    ![The End-to-end transaction displays with the HTTP Post call selected.](./media/asp-net-core/e2e-http-call.png "HTTP POST details")
+
+6. Select a SQL item to review the SQL command text issued to the database.
+
+    ![The End-to-end transaction displays with SQL command details.](./media/asp-net-core/e2e-db-call.png "SQL Command text details")
+
+7. Optionally select Dependency (outgoing) requests to Azure Storage or the Language Service.
+
+8. Return to the **Performance** screen, and select the **Dependencies** tab to investigate calls into external resources. Notice the Operations table includes calls into Sentiment Analysis, Blob Storage, and Azure SQL.
+
+    ![The Performance screen displays with the Dependencies tab selected and the Operations table highlighted.](./media/asp-net-core/performance-dependencies.png "Dependency Operations")
+
+### Sampling
+
+The Application Insights SDK for ASP.NET Core supports both fixed-rate and adaptive sampling. By default, adaptive sampling is enabled. You can disable or customize the sampling behavior.
+
+#### Disabling adaptive sampling
+
+The default sampling feature can be disabled while adding Application Insights service through a configuration option.
+
+1. With Visual Studio 2022, locate and open `Program.cs`.
+
+2. In `Program.cs` locate the `services.AddApplicationInsightsTelemetry()` code and amend it as follows to include the `ApplicationInsightsServiceOptions` object when adding the Application Insights Telemetry service.
+
+    ```csharp
+    var aiOptions = new Microsoft.ApplicationInsights.AspNetCore.Extensions.ApplicationInsightsServiceOptions();
+    aiOptions.EnableAdaptiveSampling = false;
+    builder.Services.AddApplicationInsightsTelemetry(aiOptions);
+    ```
+
+The above code will disable adaptive sampling. Follow the steps below to add sampling with more customization options.
+
+#### Configure adaptive sampling settings
```

Comment:
i think its best to remove the Configure sampling settings from this tutorial. Its not really needed, and the existing doc for it is more than sufficient, should the user decide to customize it.

Reply:
Fixed.